.github/workflows/push.yml

```diff
@@ -177,6 +177,7 @@ jobs:
               *.log
               
   dev-docs:
+    if: github.event.pull_request.merged == true
```

Comment:
This will also run for PR that are not merged in `development`.

I think is better to only run the job for the `development` branch. 

```
if: github.ref == 'refs/heads/development'
```